vertica_python/vertica/cursor.py

```diff
@@ -459,11 +452,10 @@ def format_operation_with_parameters(self, operation, parameters, is_csv=False):
         return operation
 
     def format_quote(self, param, is_csv):
-        # TODO Make sure adapt() behaves properly
         if is_csv:
             return u'"{0}"'.format(re.escape(param))
         else:
-            return QuotedString(param.encode(UTF_8, self.unicode_error)).getquoted()
+            return u"'{0}'".format(param.replace(u"'", u"''"))
```

Comment:
There are regressions with this change:
```python
cursor.execute("CREATE TABLE tbl (col_1 VARCHAR)")
cursor.execute("INSERT INTO tbl VALUES (:val)", {'val': u'\\backslash data\\'})
cursor.execute("SELECT * FROM tbl")
print(cursor.fetchall())
# Previously with psycopg2: [['\\backslash data\\']]
# Now removed psycopg2: [['\x08ackslash data\\']]
```

Reply:
Oh, in PostgreSQL's `PQescapeString`, backslashes are indeed also doubled, thanks for spotting that!